demo/OIFconfig_benchmark.ini

```diff
@@ -150,4 +150,8 @@ magnitude_decimals = 3
 # They may have unexpected results or break the code entirely.
 
 # SQL query for extracting data from the pointing database.
-pointing_sql_query = SELECT observationId, observationStartMJD, filter, seeingFwhmGeom, seeingFwhmEff, fiveSigmaDepth, fieldRA, fieldDec, rotSkyPos FROM observations order by observationId
\ No newline at end of file
+pointing_sql_query = SELECT observationId, observationStartMJD, filter, seeingFwhmGeom, seeingFwhmEff, fiveSigmaDepth, fieldRA, fieldDec, rotSkyPos FROM observations order by observationId
+
+
+lightcurve = False
+lc_model = Sinusoidal
```

Comment:
minor point - I wouldn't be shy about using a longer config name here. i.e. `lightcurve_model` would be fine in my book :)